### PR TITLE
[4.0] CPanel: Fix access rights for edit and unpublish button

### PR DESCRIPTION
**Pull request by chmst**
### Summary of Changes
On cPanel, users with appropriate permissions can edit or/and activate/deactivate Modules.
This PR fixes checks for access rights

### Testing Instructions
You need a user in a user group with following permissions:
**Cannot Edit** in com_modules but **can EditState**
You need some modules on cPanel. 
Try the coq for editiing the module or its state.

### Expected result
The Button Unpublish is visible

screen shot 2019-04-13 at 21 02 52

### Actual result
There are no buttons

### CODE REVIEW

File: administrator/templates/atum/html/layouts/chromes/well.php [hunk 1]
```diff
@@ -24,7 +24,8 @@
 	// Permission checks
 	$user    = Factory::getUser();
 	$canEdit = $user->authorise('core.edit', 'com_modules.module.' . $id) && $user->authorise('core.manage', 'com_modules');
-
+	$canChange  = $user->authorise('core.edit.state', 'com_modules.module.' . $id) && $user->authorise('core.manage', 'com_modules');
+
```
Comment (Quy) [inline administrator/templates/atum/html/layouts/chromes/well.php]: Remove tab.

**Comment by infograf768** [comment]
Look also at
https://github.com/joomla/joomla-cms/pull/24583
First merged, first served... ;)

**Comment by infograf768** [comment]
On review. I have tested this item :white_check_mark: successfully on a89d2f661da37d5469cd3376022fb91fe09aa814<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24581">issues.joomla.org/tracker/joomla-cms/24581</a>.</sub>

**Comment by wilsonge** [comment]
Thanks!

**Pull request merged**